pkg/sql/information_schema.go

```diff
@@ -743,6 +743,11 @@ func forEachTableDescWithTableLookupInternal(
 		if table, ok := desc.(*sqlbase.TableDescriptor); ok && !table.Dropped() {
 			dbName, ok := dbIDsToName[table.GetParentID()]
 			if !ok {
+				// Contrary from `crdb_internal.tables`, where dropped tables
```

Comment:
`s/Contrary from/Contrary to/`

Reply:
Also, could you give a reason for the difference in this comment? Right now you just state that a difference exists